libm-cve-2020-10029/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM debian:buster-slim
+LABEL maintainer="guido@guidovranken.com"
+
+RUN apt-get update && \
+    apt-get install -y \
+    build-essential \
+    curl \
+    gawk \
+    bison \
+    python3
+
+RUN mkdir -p /src
+RUN mkdir -p /build
+WORKDIR /src
+
+RUN curl -O https://ftp.gnu.org/gnu/libc/glibc-2.31.tar.bz2 && \
+    tar jxf glibc-2.31.tar.bz2
+RUN mkdir /src/glibc-2.31/build
+
+COPY src/* /build/
+RUN /build/build.sh
```

Comment:
If possible, could we use a builder container to reduce the final container size?

Reply:
How do you suggest we do that?

Reply:
After the top FROM you can specify "as builder" then you can do another FROM debian and COPY --from=builder to copy over the binary.

https://docs.docker.com/develop/develop-images/multistage-build/

Reply:
Done.